openmetadata-ui/src/main/resources/ui/src/utils/SvgUtils.tsx

```diff
@@ -130,6 +130,7 @@ import Logo from '../assets/svg/logo.svg';
 import IconManageColor from '../assets/svg/manage-color.svg';
 import IconMinus from '../assets/svg/minus.svg';
 import IconMlModal from '../assets/svg/mlmodal.svg';
+import IconMSTeamsGrey from '../assets/svg/ms-teams-grey.svg';
```

Comment:
are we using `ms-tems` icon that we added?

Reply:
We were using it just not added her. Updating here also.